### ISSUE TITLE
Laminas\Mvc\Plugin\FlashMessenger\View\Helper don't use Laminas\Mvc\Plugin\FlashMessenger\FlashMessenger

### ISSUE THREAD

**Issue by basselin** [laminas/laminas-mvc-plugin-flashmessenger#27]
### BC Break Report

|    Q        |   A
|------------ | ------
| Version     | 1.8.0

#### Summary

`Laminas\Mvc\Plugin\FlashMessenger\View\Helper\FlashMessenger` don't use `Laminas\Mvc\Plugin\FlashMessenger\FlashMessenger` without config `['view_helper_config']['flashmessenger']`.

#### Previous behavior

https://github.com/laminas/laminas-mvc-plugin-flashmessenger/blob/420a99df81432140d78799a2e7cb34931fcdd619/src/View/Helper/FlashMessengerFactory.php#L32

#### Current behavior

https://github.com/laminas/laminas-mvc-plugin-flashmessenger/blob/86bb1c654e5fafaf34d4deaa7dc9af721fbcb42d/src/View/Helper/FlashMessengerFactory.php#L47

#### How to reproduce

Config without `['view_helper_config']['flashmessenger']`

#### Solution in FlashMessengerFactory ?
```php
$helper = new FlashMessenger();
$helper->setPluginFlashMessenger($flashMessenger);
return helper;
```
Or in global Config :
```php
[
    'view_helper_config' => [
        'flashmessenger' => [],
    ],
]
```

**Comment by Ocramius** [comment]
Hey @basselin, could you rephrase a bit? What is broken?

**Comment by froschdesign** [comment]
> #### Solution in FlashMessengerFactory ?
> ```
> $helper = new FlashMessenger();
> $helper->setPluginFlashMessenger($flashMessenger);
> return helper;
> ```

This is the correct solution.

**Comment by basselin** [comment]
Hi @Ocramius,
The helper `flashMessenger` don't use the plugin service `'flashMessenger'` if you don't have any configuration of "flashmessenger" in "view_helper_config" : `['view_helper_config']['flashmessenger']`

Therefore, the messages no longer appear.
